[2025-02-19T02:08:10.880+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingestion_pipeline.write_success_flag manual__2025-02-19T02:08:07.971516+00:00 [queued]>
[2025-02-19T02:08:10.887+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingestion_pipeline.write_success_flag manual__2025-02-19T02:08:07.971516+00:00 [queued]>
[2025-02-19T02:08:10.887+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2025-02-19T02:08:10.897+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): write_success_flag> on 2025-02-19 02:08:07.971516+00:00
[2025-02-19T02:08:10.901+0000] {standard_task_runner.py:57} INFO - Started process 534 to run task
[2025-02-19T02:08:10.903+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ingestion_pipeline', 'write_success_flag', 'manual__2025-02-19T02:08:07.971516+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_pipeline.py', '--cfg-path', '/tmp/tmpjctjdv1l']
[2025-02-19T02:08:10.905+0000] {standard_task_runner.py:85} INFO - Job 114: Subtask write_success_flag
[2025-02-19T02:08:10.936+0000] {task_command.py:410} INFO - Running <TaskInstance: ingestion_pipeline.write_success_flag manual__2025-02-19T02:08:07.971516+00:00 [running]> on host 29612c003465
[2025-02-19T02:08:10.995+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ingestion_pipeline' AIRFLOW_CTX_TASK_ID='write_success_flag' AIRFLOW_CTX_EXECUTION_DATE='2025-02-19T02:08:07.971516+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-19T02:08:07.971516+00:00'
[2025-02-19T02:08:10.997+0000] {logging_mixin.py:150} INFO - Success flag written to /opt/***/data/ingestion/_SUCCESS
[2025-02-19T02:08:10.997+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-02-19T02:08:11.004+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=ingestion_pipeline, task_id=write_success_flag, execution_date=20250219T020807, start_date=20250219T020810, end_date=20250219T020811
[2025-02-19T02:08:11.051+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-02-19T02:08:11.065+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
